package.json

```diff
@@ -13,6 +13,19 @@
     "android": "react-native run-android",
     "snyk-protect": "snyk protect"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint . --fix",
```

Comment:
in windows this always fails due to 		https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/5aec0ec1860b32158f4c69bd16cca14fa34c70f6/.eslintrc.js#L121 , So I dont think its a good idea

Reply:
That's interesting about windows 🤔
Do you agree that running `"eslint .",` and `"jest ./__tests__/*",` in pre-commit is useful and will save lots of time and some nerves? 🙂

Reply:
yes , it is but eslint --fix don't solve all the problems. and the one setting I mentioned cannot be solved In windows,  you can read about it (https://stackoverflow.com/questions/37826449/expected-linebreaks-to-be-lf-but-found-crlf-linebreak-style) if we can change this setting , which i don't think this PR is ok

Reply:
I removed `eslint --fix` and `git add` as you suggested. So now it's just running eslint and tests in precommit.

Reply:
yaah, but still eslint is going to fail and person using windows will not be able to push commits

Reply:
Thanks for checking again!
Hopefully this one will be the last edition of this PR :)
I disabled linebreak-style rule in eslint as it's not important. More important is to not push 💩 code in repo...